compute_tools/src/http/routes/prewarm_lfc.rs

```diff
@@ -0,0 +1,214 @@
+use crate::compute::ComputeNode;
```

Comment:
As stated in a previous comment, please move all actual logic into the `ComputeNode` struct and just have the handlers call those functions.